samples_project/Assets/SampleViewer/Samples/VRSample/Scripts/ContinuousMovement.cs

```diff
@@ -15,15 +15,25 @@ public class ContinuousMovement : MonoBehaviour
     private CharacterController controller;
     [SerializeField] private float speed;
     [SerializeField] private float upSpeed;
+    public bool useSmoothTurn = true;
```

Comment:
I suggest making this a property. You can call `ToggleSnapTurn` on set.
Also, capitalize the public property.